packages/sdk-js/src/GrowthBook.ts

```diff
@@ -129,10 +133,48 @@ export class GrowthBook<
   }
 
   public getApiInfo(): [ApiHost, ClientKey] {
-    return [
-      (this._ctx.apiHost || "https://cdn.growthbook.io").replace(/\/*$/, ""),
-      this._ctx.clientKey || "",
-    ];
+    return [this.getApiHosts().apiHost, this.getClientKey()];
+  }
+  public getApiHosts(): {
+    apiHost: string;
+    streamingHost: string;
+    remoteEvalHost: string;
+    featuresPath: string;
+    streamingPath: string;
+    remoteEvalPath: string;
+    apiRequestHeaders: Record<string, string>;
+  } {
+    const defaultHost = this._ctx.apiHost || "https://cdn.growthbook.io";
+    return {
+      apiHost: defaultHost.replace(/\/*$/, ""),
+      streamingHost: (this._ctx.streamingHost || defaultHost).replace(
+        /\/*$/,
+        ""
+      ),
+      remoteEvalHost: (this._ctx.remoteEvalHost || defaultHost).replace(
+        /\/*$/,
+        ""
+      ),
+      featuresPath: this._ctx.featuresPath || "/api/features",
+      streamingPath: this._ctx.streamingPath || "/sub",
+      remoteEvalPath: this._ctx.remoteEvalPath || "/eval",
+      apiRequestHeaders: this._ctx.apiRequestHeaders || {},
+    };
+  }
+  public getClientKey(): string {
+    return this._ctx.clientKey || "";
+  }
+
+  public getRemoteEval(): boolean {
```

Comment:
Naming wise, should this be `isRemoteEval` or something instead to make it more clear it's a toggle?